tools/version_check.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
```

Comment:
I could be mistaken, but does this constitute us adding python3 as a dependency to build the toolkit. 
I believe that technically this is already a transitive requirement as we require gcloud and gcloud requires python3, but python is not listed in our dependencies: https://cloud.google.com/hpc-toolkit/docs/setup/install-dependencies

Since we already have the transitive dependency, I would not block on this, but it may be good to have @cboneti weigh in.